pkg/virt-controller/watch/vmi.go

```diff
@@ -2452,3 +2454,48 @@ func (c *VMIController) syncMemoryHotplug(vmi *virtv1.VirtualMachineInstance) {
 		vmi.Labels[virtv1.MemoryHotplugOverheadRatioLabel] = *overheadRatio
 	}
 }
+
+func (c *VMIController) aggregateDataVolumesConditions(vmiCopy *virtv1.VirtualMachineInstance, dvs []*cdiv1.DataVolume) {
+	if len(dvs) == 0 {
+		return
+	}
+
+	dvsReadyCondition := virtv1.VirtualMachineInstanceCondition{
+		Status:  k8sv1.ConditionTrue,
+		Type:    virtv1.VirtualMachineInstanceDataVolumesReady,
+		Reason:  virtv1.VirtualMachineInstanceReasonAllDVsReady,
+		Message: "All of the VMI's DVs are bound and not running",
+	}
+
+	var dvMessages []string
+	for _, dv := range dvs {
+		cStatus, cMessage := statusAndMessageOfReadyCondition(dv.Status.Conditions)
```

Comment:
The message is not used anymore:
```suggestion
		cStatus := statusOfReadyCondition(dv.Status.Conditions)
```
making the `dvMessages` variable useless too.

Reply:
done